libs/remix-ui/solidity-uml-gen/src/lib/solidity-uml-gen.tsx

```diff
@@ -62,16 +74,22 @@ export function RemixUiSolidityUmlGen ({ plugin, updatedSvg, loading, themeSelec
       buttonText: 'Download as PDF',
       svgValid: () => validSvg,
       action: () => console.log('generated!!'),
-      icon: 'fa mr-1 pt-1 pb-1 fa-file'
+      icon: 'fa fa-file'
     },
     { 
       buttonText: 'Download as PNG',
       svgValid: () => validSvg,
-      action: () => console.log('generated!!'),
+      action: () => console.log('generated!!!'),
```

Comment:
Action can be a success message displayed in tooltip.

Reply:
Actually action is supposed to be a void function that will activate either download of a pdf or a png. These are place holders for now.